public/assets/images/pages/calendar.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64">
```

Comment:
let's create react components for this icons and put them in:
```src/components/icons/IconName/index.js```
and add an index.js ininside icons folder that exports all icons like this:
```
import ChartIcon from 'Chart';
import PersonIcon from 'Person';

export {
    ChartIcon,
    PersonIcon,
};
```